xep-0369.xml

```diff
@@ -80,11 +85,11 @@
 
 <section1 topic='Requirements' anchor='reqs'>
   <ul>
-    <li>A user's participation in a channel persists and is not modified by the user's client going online and offline.</li>
+    <li>A user's participation in a channel may persist and is not modified by the user's client going online and offline.</li>
     <li>Multiple devices associated with the same account can share the same nick in the channel, with well-defined rules making each client individually addressable.</li>
     <li>Channels are not required to support or reflect presence for participants.</li>
     <li>A reconnecting client can quickly resync with respect to messages and presence.</li>
-    <li>A user may (subject to configuration) receive messages from a channel as an invisible observer.</li>
+    <li>A user may (subject to server policy) receive messages from a channel as an invisible observer.</li>
```

Comment:
I think configuration covers both server-wide and room configuration, while server policy seems to preclude servers allowing but rooms not.